frontend/static.json

```diff
@@ -0,0 +1,11 @@
+{
+  "root": "build/",
+  "routes": {
+    "/**": "index.html"
+  },
+  "proxies": {
+    "/api/": {
+      "origin": "${API_URL}"
+    }
+  }
+}
```

Comment:
Might be good to document in the wiki about how Heroku is set up (what buildpacks or settings) that will read this file.